zingolib/src/wallet/sync.rs

```diff
@@ -0,0 +1,35 @@
+//! Trait implmentations for sync interface
+
+use std::collections::HashMap;
+
+use zcash_keys::keys::{UnifiedFullViewingKey, UnifiedSpendingKey};
+use zingo_sync::interface::SyncWallet;
+use zip32::AccountId;
+
+use crate::wallet::LightWallet;
+
+impl SyncWallet for LightWallet {
+    type Error = ();
+
+    fn get_unified_full_viewing_keys(
```

Comment:
nice logic! i look forward to merging this as the usk.

Reply:
thanks!